packages/components/inputs/select-input/src/select-input.visualroute.jsx

```diff
@@ -113,6 +122,16 @@ const DefaultRoute = () => (
         horizontalConstraint={7}
       />
     </Spec>
+    <Spec label="with multiple values selected and condensed">
+      <SelectInput
+        value={['one', 'two']}
+        onChange={() => {}}
+        options={options}
+        isMulti={true}
+        isCondensed={true}
+        horizontalConstraint={7}
+      />
+    </Spec>
```

Comment:
It appears that the height of the multivalue wrapper is set to `32px`, which prevents us from seeing the condensed padding.
| Current  | Expected  |
|---|---|
| <img width="625" alt="Screenshot 2024-04-15 at 09 23 04" src="https://github.com/commercetools/ui-kit/assets/49066275/91f6f588-c1b3-4d7d-b8f2-dedb2c83cd15">  | <img width="399" alt="Screenshot 2024-04-15 at 09 25 59" src="https://github.com/commercetools/ui-kit/assets/49066275/823b2107-1bcf-4726-80c5-e6f71a32248e">  |

Reply:
Fixed in [this commit](https://github.com/commercetools/ui-kit/pull/2774/commits/bf8794eff8a421d3f6c7ac8a4a06fe13827605ef)